app/models/order.rb

```diff
@@ -0,0 +1,3 @@
+class Order < ActiveRecord::Base
+    has_many :order_items
+end
```

Comment:
You may want to have more validations on order, like a credit information must be present, etc. Also any calculation logic can go in this model file.